dist/t/spec/support/capybara.rb

```diff
@@ -24,7 +24,7 @@
 
 # Set hostname
 begin
-  hostname = Socket.gethostbyname(Socket.gethostname).first
+  hostname = Addrinfo.getaddrinfo(Socket.gethostname).first
```

Comment:
With that change, we could also get rid of these lines in the RuboCop TODO file: https://github.com/openSUSE/open-build-service/blob/0a61c6e93b22936bf36864e2f999d84a29318ba5/.rubocop_todo.yml#L27-L32

Reply:
In irb, this results in `ArgumentError (wrong number of arguments (given 1, expected 2..6))`
and the output format is different, too.

`Addrinfo.getaddrinfo(Socket.gethostname, 443, nil, :STREAM).first.getnameinfo[0]` also returns a name.